neurovault/urls.py

```diff
@@ -409,6 +419,21 @@ def images(self, request, pk=None):
             page, context={'request': request}, many=True)
         return paginator.get_paginated_response(serializer.data)
 
+    def add_image(self, request, pk=None):
+        collection = get_collection(pk, request, mode='api')
+
+        if not owner_or_contrib(request, collection):
+            self.permission_denied(request)
+
+        image = StatisticMap(collection=collection)
+        serializer = EditableStatisticMapSerializer(data=request.data,
+                                                    instance=image)
```

Comment:
To clarify, the URL used for posting images in this PR is 

    http://neurovault.org/api/collections/<id>/images/

So it's nested under collection endpoint and gets the collection id from the url itself. Do you want me to change it to

    http://neurovault.org/api/images/

Along with new endpoints, or to make these new endpoints nested under the collection URL:

    http://neurovault.org/api/collections/<id>/atlases/
    http://neurovault.org/api/collections/<id>/nidm_results/

It looks like the latter is faster to implement since it'll require less changes.